scripts/unit_test/test_usb_reset_mount.py

```diff
@@ -0,0 +1,19 @@
+"""scripts/unit_test/test_usb_reset_mount.py: Test usb_reset.mount and .umount"""
+from __future__ import print_function
+
+import sys
+
+import pytest
```

Comment:
Correct me if I'm wrong, but I believe pytest is not available in xenserver 8. Is this means to be run when building the xapi package?

Reply:
I was looking for invocation of Python unittest in the xapi build and could not find one.

I checked if the tests would pass using python -m unittest and found this:

In scripts/test_usb_scan.py, the decorator `@nottest` is used to declare some functions as not being tests.

But Python `unittest` does not know about @nottest and tries to execute these functions as if they were tests, which fails:
```py
# (cd scripts;python2 -m unittest discover 2>&1 |grep ERROR)
# (cd scripts;python3 -m unittest discover 2>&1 |grep ERROR)
ERROR: test_usb_common (test_usb_scan.TestUsbScan)
ERROR: test_usb_config_error_common (test_usb_scan.TestUsbScan)
ERROR: test_usb_dongle (test_usb_scan.TestUsbScan)
ERROR: test_usb_dongle_on_hub (test_usb_scan.TestUsbScan)
ERROR: test_usb_dongle_unbinded (test_usb_scan.TestUsbScan)
ERROR: test_usb_exit (test_usb_scan.TestUsbScan)
ERROR: test_usb_keyboard (test_usb_scan.TestUsbScan)
```

I concluded that there is no way that the unit tests are executed in the xapi build if `pytest` (or the formerly used nosetest) is not available.